contribs/gmf/src/directives/partials/authentication.html

```diff
@@ -0,0 +1,165 @@
+<div ng-if="gmfUser.username">
+  <div class="form-group">
+    <span>{{ authCtrl.translate('Logged in as') }}</span>
+    <strong>{{ gmfUser.username }}</strong>.
```

Comment:
Okay, so with `::`, i.e. one time binding, the username is properly set.  That means I don't fully understand what the one time binding truly does...

I'll change this to one time binding, as requested.